.github/workflows/build-and-push-helm.yaml

```diff
@@ -2,14 +2,23 @@ name: Match chart version and push to registry
 
 on:
   workflow_call:
+    inputs:
+      scope:
```

Comment:
@AnguIar Please change to choice input type with a pre-defined options
see an example here: https://tkolber.medium.com/dropdown-input-choice-workflows-in-github-actions-e95cf140bf61